mobile-app/app/hooks/useUnitSuffix.ts

```diff
@@ -0,0 +1,17 @@
+import BigNumber from 'bignumber.js'
+
+export function useUnitSuffix (units: Record<number, string>, value?: string): string | undefined {
+  if (value === undefined) {
```

Comment:
Let move this checking to the code that calls this hook, so we can change `value` to a not an optional prop and always return `string`